recipe/meta.yaml

```diff
@@ -95,21 +96,27 @@ test:
     {% endfor %}
 
     # Test ncurses library arguments.
-    - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib .*-lncurses -ltinfo$"
-    - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib .*-ltinfo$"
-    - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib .*-lncursesw -ltinfow$"
-    - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib .*-ltinfow$"
-
-    # Usage tests
-    # - bash run_test.sh    # [osx]
-    # Python test will not pass until it is recompiled against ncurses 6.1
-    # - python run_test.py  # [osx]
+    # - pkg-config ncursesw --libs | grep "^-L${PREFIX}/lib -lncursesw -ltinfow$"
+    # - pkg-config tinfow --libs | grep "^-L${PREFIX}/lib -ltinfow$"
+    # - pkg-config ncurses --libs | grep "^-L${PREFIX}/lib -lncurses -ltinfo$"
+    # - pkg-config tinfo --libs | grep "^-L${PREFIX}/lib -ltinfo$"
```

Comment:
Why are we dropping the `pkg-config` tests? These were added to satisfy Bioconda's needs. :/